File: 251101/색종이의 총 넓이/the-total-area-of-colored-paper.py
offset = 200;

arr = [
    [0] * offset for i in range (offset)
]

n = int(input())

# 가로 세로 넓이가 8임.

for i in range (0, n) :
    x, y = tuple(map(int, input().split()))
    x += (offset // 2)
    y += (offset // 2)

    for j in range (x, x+8) :
        for k in range (y, y+8) :
            arr[j][k] = 1;


answer = 0;

for i in range (0, offset) :
    for j in range (0, offset) :
        if arr[i][j] == 1 :
            answer += 1;

print(answer)

File: 251101/색종이의 총 넓이/the-total-area-of-colored-paper.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        // Please write your code here.

        //가로 세로 8,
        //offset

        int offset = 100;

        int arr [][] = new int[offset * 2][offset * 2];

        Scanner sc = new Scanner(System.in);

        int num = sc.nextInt();

        for(int i = 0; i < num; i++) {
            int x = sc.nextInt() + offset;
            int y = sc.nextInt() + offset;

            for(int j = x; j < x+8; j++){
                for(int k = y; k < y+8; k++){
                    arr[j][k] = 1;
                }
            }

        }

        int answer = 0;

        for(int i = 0; i < offset*2; i++){
            for(int j =0; j < offset*2; j++){
                if(arr[i][j] == 1) answer++;
            }
        }



        System.out.println(answer);


    }
}